pkg/terraform/parser/parser_test.go

```diff
@@ -209,7 +213,11 @@ func TestParser_ProcessStates(t *testing.T) {
 				t.Errorf("ProcessStates() failed to read json file %v", err)
 			}
 
-			gcsClient := &storage.MockStorageClient{GetData: string(data), GetErr: tc.wantErr}
+			gcsClient := &storage.MockStorageClient{
+				GetData:       string(data),
+				GetErr:        tc.wantErr,
+				GetCancelFunc: func() {},
```

Comment:
nit: can we rename these `Get*` -> `Download*`

Reply:
Done